template/python/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/polyxia-org/alpha:main as get-alpha
+FROM --platform=${TARGETPLATFORM:-linux/amd64} python:3-alpine
+
+ARG TARGETPLATFORM
+ARG ADDITIONAL_PACKAGE
+
+RUN apk --no-cache add curl ca-certificates openrc util-linux ${ADDITIONAL_PACKAGE} && \
+    addgroup -S runner && adduser -S -g runner runner
+
+# Get Alpha and make it executable in the final build image
+COPY --from=get-alpha /openrc.sh /etc/init.d/alpha
+COPY --from=get-alpha /alpha /usr/bin/alpha
+RUN chmod +x /usr/bin/alpha && chmod +x /etc/init.d/alpha
+
+# Temporary. Just for debugging purposes. Without the following lines, we 
+# can't enter the microVM through a terminal.
+RUN ln -s agetty /etc/init.d/agetty.ttyS0
+RUN echo ttyS0 >/etc/securetty
+RUN rc-update add agetty.ttyS0 default
+RUN echo "root:root" | chpasswd
+
+# Make alpha start on boot
+RUN rc-update add alpha default
+RUN echo 'export ALPHA_INVOKE="python /home/runner/index.py"' >> /etc/conf.d/alpha && \
+    echo 'export PYTHONPATH="/home/runner/.local/lib/python3.11/site-packages"' >> /etc/conf.d/alpha
+
+USER runner
+
+RUN mkdir -p /home/runner/function
+
+WORKDIR /home/runner/
+
+COPY --chown=runner:runner index.py           .
+COPY --chown=runner:runner requirements.txt   .
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN touch ./function/__init__.py
+WORKDIR /home/runner/function/
+COPY --chown=runner:runner function/requirements.txt	.
+RUN pip install --no-cache-dir -r requirements.txt
+
+# install function code
+COPY --chown=runner:runner function/   .
```

Comment:
`PORT` is missing to document the Dockerfile

Reply:
We don't care of this, as we don't build real docker images. We use Dockerfile only because it is more easier to build a rootfs to after extract it from a container, but some Docker directives (`ENV`, `PORT`, `ENTRYPOINT` ...) are not useful here because we extract the rootfs only.

All those directives are saved by Docker into the OCI `config.json` file. But here, we don't want it.

If we need to document the port, it should be done more generally as the port will be the same across runtimes.